prime-router/src/main/kotlin/fhirengine/translation/hl7/FhirToHl7Converter.kt

```diff
@@ -98,6 +106,7 @@ class FhirToHl7Converter(
         // Add any schema level constants to the context
         // We need to create a new context, so constants exist only within their specific schema tree
         val schemaContext = CustomContext.addConstants(schema.constants, context)
+
```

Comment:
Is there some kind of standard for these kinds of formatting changes on the team? Have we discussed using a common formatter/linter?

Reply:
We're using klinter with some standard settings that are stored....somewhere. On Intellij you should get lots of notices after installing the plugins